src/darwin/Framework/CHIP/templates/CHIPClustersObjc-src.zapt

```diff
@@ -37,7 +37,7 @@ using namespace chip::app::Clusters;
 {{/unless}}
 - (void){{asLowerCamelCase name}}WithParams: (CHIP{{asUpperCamelCase parent.name}}Cluster{{asUpperCamelCase name}}Params * {{#unless (commandHasRequiredField .)}}_Nullable{{/unless}})params completionHandler:({{>command_completion_type command=.}})completionHandler
 {
-    chip::Optional<uint16_t> timedInvokeTimeoutMs;
+    chip::Optional<uint16_t> timedInvokeTimeoutMs = chip::NullOptional;
```

Comment:
Fwiw, NullOptional is what the default constructor constructs to.

Reply:
Ok. Let me remove this. Thank you.

Reply:
Updated